src/cascade/executor/epiviz_runner.py

```diff
@@ -131,6 +158,10 @@ def model_context_from_settings(execution_context, settings):
     add_mortality_data(model_context, execution_context)
     add_omega_constraint(model_context, execution_context)
     model_context.average_integrand_cases = make_average_integrand_cases_from_gbd(execution_context)
+    add_special_study_covariates(study_covariate_records, bundle, model_context.average_integrand_cases)
+
+    fixed_effects_from_epiviz(model_context, study_covariate_records, execution_context, settings)
+    random_effects_from_epiviz(model_context, settings)
```

Comment:
Seeing these (FE's and RE's) makes me wonder if there is a model_context.fixed_effects and model_context.random_effects?  Maybe add details to the comments above Steps 6 and 7.

Reply:
The definition of fixed and random effects isn't strong in statistics literature, and it's not strong here. I'm pushing for representations in the Model object that say what the moving parts are (covariate, multiplier, priors). The moment we call them fixed and random should be when changing to the dismod file. Maybe this distinction is better put lower in the code. It's good that it's at the end of the translation process. We're not far off in terms of representation.